internal/cmd/group.go

```diff
@@ -25,8 +25,48 @@ func init() {
 	addCanaryFlag(groupsCreateCmd)
 	groupCmd.AddCommand(groupsDestroyCmd)
 	addYesFlag(groupsDestroyCmd, "Confirms the destruction of the group, with all its locations and databases.")
+	groupCmd.AddCommand(groupShowCmd)
 }
 
+var groupShowCmd = &cobra.Command{
+	Use:               "show <group-name>",
+	Short:             "Show information about a group.",
+	Args:              cobra.ExactArgs(1),
+	ValidArgsFunction: dbNameArg,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		cmd.SilenceUsage = true
+		client, err := authedTursoClient()
+		if err != nil {
+			return err
+		}
+
+		name := args[0]
+
+		databases, err := client.Databases.List()
+		if err != nil {
+			return err
+		}
+
+		group, err := getGroup(client, name)
+		if err != nil {
+			fmt.Printf("group %s not found\n", internal.Emph(name))
+			return nil
+		}
+
+		version := "unknown"
+		for _, db := range databases {
+			if name == db.Group {
+				if db.Version != "" {
+					version = db.Version
```

Comment:
We should get the version from the API, let me get that fixed for ya

Reply:
done